dr-solution-introduction.md

```diff
@@ -0,0 +1,117 @@
+---
+title: TiDB DR Solution Introduction
+summary: Learn about the disaster solutions provided by TiDB, including disaster recovery based on the primary and secondary clusters, disaster recovery based on multiple replicas in a single cluster, and disaster recovery based on backup and restore.
+---
+
+# TiDB DR Solution Introduction
+
+This document introduces the disaster recovery (DR) solutions provided by TiDB. The organization of this document is as follows:
+
+- Describe basic DR concepts.
+- Introduce the architecture of TiDB, TiCDC, and Backup & Restore (BR).
+- Describes the DR solutions provided by TiDB.
+- Compare these DR solutions.
+
+## Basic concepts
+
+- RTO (Recovery Time Objective): Time required for the system to recover from a disaster.
+- RPO (Recovery Point Objective): The maximum amount of data loss that business can tolerate in the event of a disaster.
+
+The following figure illustrates these two concepts:
+
+![RTO and RPO](/media/dr/rto-rpo.png)
+
+- Error tolerance objective: Because a disaster can affect different regions. In this document, the term error tolerance objective is used to describe the maximum impact range of a disaster that the system can tolerate.
```

Comment:
```suggestion
- Error tolerance objective: Because a disaster can affect different regions. In this document, the term error tolerance objective is used to describe the maximum range of a disaster that the system can tolerate.
```